_data/simple-icons.json

```diff
@@ -560,6 +560,11 @@
             "hex": "E53238",
             "source": "https://go.developer.ebay.com/logos"
         },
+        {
+            "title": "Eclipse",
+            "hex": "F7941E",
```

Comment:
The color associated with the Eclipse IDE seems to be the purple `#2C2255` (taken from the text in the Eclipse IDE SVG), with the orange `#F7941E` being primarily for accents. The Eclipse Foundation uses the orange as their primary color.